# I don't actually know that this is necessary, but Fedora and openmandriva both carry it

diff -up qtbase-everywhere-src-5.14.2/src/corelib/global/qlibraryinfo.cpp.no_relocatable qtbase-everywhere-src-5.14.2/src/corelib/global/qlibraryinfo.cpp
--- qtbase-everywhere-src-5.14.2/src/corelib/global/qlibraryinfo.cpp.no_relocatable	2020-03-27 04:49:31.000000000 -0500
+++ qtbase-everywhere-src-5.14.2/src/corelib/global/qlibraryinfo.cpp	2020-04-13 15:13:44.075705226 -0500
@@ -671,8 +671,11 @@ static QString getPrefix(
 #  if QT_CONFIGURE_CROSSBUILD
     if (group == QLibraryInfo::DevicePaths)
         return QString::fromLocal8Bit(QT_CONFIGURE_PREFIX_PATH);
-#  endif
+#  elif 0 //QT_CONFIG(relocatable)
     return getExtPrefixFromHostBinDir();
+#  else
+    return QString::fromLocal8Bit(QT_CONFIGURE_PREFIX_PATH);
+#  endif
 #elif QT_CONFIG(relocatable)
     return getRelocatablePrefix();
 #else
